ui/allofus-anvil-imputation/index.html

```diff
@@ -42,13 +42,10 @@ <h1 class="heading-title"><i>All of Us</i> + AnVIL Imputation Service</h1>
     <img id="dna-orbit" src="img/dna-orbit.png" alt="DNA orbit icon" />
     <div class="frame-2-text">
       <div class="beta-service">
-        Beta service launching<br/>
-        summer 2025
+        Beta is now available
       </div>
       <div class="sign-up">
-        Interested in staying informed about our imputation service?<br/>
-        <a href="mailto:scientific-services-support@broadinstitute.org?subject=I%20would%20like%20to%20hear%20when%20the%20imputation%20service%20opens">Sign up</a>
-        and we will let you know when we launch.
+        <a href="https://broadscientificservices.zendesk.com/hc/en-us/articles/39901941351323" target="_blank">Get started</a>
```

Comment:
`target="_blank"` opens the link in a new window/tab, right? that's a great call